packages/instantsearch.js/src/widgets/chat/chat.tsx

```diff
@@ -28,12 +28,17 @@ import type {
   TemplateWithBindEvent,
   BaseHit,
 } from '../../types';
-import type { ChatClassNames, Tools, VNode } from 'instantsearch-ui-components';
+import type {
+  ChatClassNames,
+  RecordWithObjectID,
+  Tools,
+  VNode,
+} from 'instantsearch-ui-components';
 
 type ItemComponent<THit> = (props: {
-  item: THit;
-  onClick: () => void;
-  onAuxClick: () => void;
+  item: RecordWithObjectID<THit>;
```

Comment:
should this not be Hit or AlgoliaHit? or do you want to explicitly make it available for recommend / item without __position too?